src/pages/nft/class/_classId/_nftId.vue

```diff
@@ -114,6 +114,11 @@
                 @collect="handleCollectFromPreviewSection"
                 @view-content="handleViewContent"
               />
+              <NFTFeatured
+                :class-id="classId"
+                :preset="nftCollectorWalletAddress && nftCollectorWalletAddress.includes(getAddress) ? 'edit' : 'invisible'"
```

Comment:
confusing preset naming, can we v-if this whole component instead inside if it needs to be hidden?

Reply:
Modified with the `readOnly` prop, since some CSS classes and props should reference it.